### ISSUE TITLE
Absolute factorization docstring example failing

### ISSUE THREAD

**Issue by thofma** [oscar-system/Oscar.jl#420]
```julia
julia> Qa, a = PolynomialRing(QQ, :a=>1:2);

julia> R, X = PolynomialRing(FractionField(Qa), :X=>1:2);

julia> f = (X[1]^2+a[1]*X[2]^2)*(X[1]+X[2]);

julia> z = factor_absolute(f)
ERROR: DomainError with 16777216:
Characteristic must be prime
Stacktrace:
 [1] GF(::Int64; cached::Bool) at /home/bla/.julia/packages/Nemo/BMnm6/src/flint/gfp_elem.jl:422
 [2] HenselCtxFqRelSeries at /home/bla/.julia/packages/Hecke/cGa6u/src/NumField/NfAbs/MPolyAbsFact.jl:106 [inlined]
 [3] macro expansion at /home/bla/.julia/packages/Hecke/cGa6u/src/Hecke.jl:588 [inlined]
 [4] Hecke.MPolyFact.RootCtx(::fmpq_mpoly, ::Int64, ::Int64, ::Int64) at /home/bla/.julia/packages/Hecke/cGa6u/src/NumField/NfAbs/MPolyAbsFact.jl:498
 [5] RootCtx at /home/bla/.julia/packages/Hecke/cGa6u/src/NumField/NfAbs/MPolyAbsFact.jl:494 [inlined]
 [6] roots(::fmpq_mpoly, ::Int64; pr::Int64) at /home/bla/.julia/packages/Hecke/cGa6u/src/NumField/NfAbs/MPolyAbsFact.jl:592
 [7] roots at /home/bla/.julia/packages/Hecke/cGa6u/src/NumField/NfAbs/MPolyAbsFact.jl:552 [inlined]
 [8] macro expansion at /home/bla/.julia/packages/Hecke/cGa6u/src/Hecke.jl:588 [inlined]
 [9] absolute_bivariate_factorisation(::fmpq_mpoly) at /home/bla/.julia/packages/Hecke/cGa6u/src/NumField/NfAbs/MPolyAbsFact.jl:1153
 [10] absolute_multivariate_factorisation(::fmpq_mpoly) at /home/bla/.julia/packages/Hecke/cGa6u/src/NumField/NfAbs/MPolyAbsFact.jl:1332
 [11] macro expansion at /home/bla/.julia/packages/Hecke/cGa6u/src/Hecke.jl:588 [inlined]
 [12] factor_absolute(::fmpq_mpoly) at /home/bla/.julia/packages/Hecke/cGa6u/src/NumField/NfAbs/MPolyAbsFact.jl:1440
 [13] afact(::fmpq_mpoly, ::Array{Int64,1}; int::Bool) at /home/bla/.julia/dev/Oscar/experimental/ModStd/ModStdQt.jl:653
 [14] afact at /home/bla/.julia/dev/Oscar/experimental/ModStd/ModStdQt.jl:595 [inlined]
 [15] factor_absolute(::AbstractAlgebra.Generic.MPoly{AbstractAlgebra.Generic.Frac{fmpq_mpoly}}) at /home/bla/.julia/dev/Oscar/experimental/ModStd/ModStdQt.jl:554
 [16] top-level scope at REPL[17]:1
```
@fieker

**Comment by fieker** [comment]
Should have been fixed with the recent AbsFact in hecke

**Comment by thofma** [comment]
Ah, ok, let me try again.

**Comment by thofma** [comment]
Right, will be fixed by #423.
